autotest.pm

```diff
@@ -45,7 +45,8 @@ loadtest is called.
 =cut
 
 sub find_script ($script) {
-    if (defined(my $wheel = (glob Cwd::getcwd . "/*/tests/$script"))) {
+    my $wheels_dir = $bmwqemu::vars{WHEELS_DIR} // Cwd::getcwd . "/*/tests/";
```

Comment:
Is this variable used anywhere?

Reply:
That's s not exactly what I suggested.
You append `"/*/tests/"` to the cwd, and one line later you append `"/*/tests/$script"` again to that string.
It only doesn't complain because we don't have a test case where the wheel would be in cwd.

Reply:
I added a test that I think should reproduce it. Unfortunately I can't get it to fail with the wrong code in `find_script` 🤔 I'm pushing it anyway, maybe you have an idea

Reply:
See https://github.com/os-autoinst/os-autoinst/pull/2356#discussion_r1317005147